texastribune/donations-app
# Permanently use single-slide widget

#### What's this PR do?
+ Kills `campaign2.html` since our Optimize experiment is over
+ Puts the single-slide widget on the main page
+ Cleans up a bunch of cruft related to that

#### Why are we doing this? How does it help us?
The single-slide variant slightly outperformed the carousel during FMD. Plus, the code behind it is much cleaner.

#### How should this be manually tested?
+ `make` and confirm no errors
+ `yarn run dev`
+ View the app at `http://localhost:4567/` and make sure the widget works. Try changing from monthly to yearly, entering manual amounts, entering invalid manual amounts, etc. Also click through the nav and confirm no links are broken. Lastly, go to `/campaign.html` and confirm it redirects to home.
+ Exit from the dev server and do `yarn run build`. Confirm you get a `build/` directory containing everything needed for the app. Open up `index.html` and make sure the page works.

#### Have automated tests been added?
No.

#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?
No.

#### What are the relevant tickets?
https://3.basecamp.com/3098728/buckets/736178/todos/745822877

#### How should this change be communicated to end users?
NA.

#### Next steps?
NA.

#### Smells?
NA.

#### TODOs:
* [ ] Make sure Rebecca is cool with this

#### Has the relevant documentation/wiki been updated?
NA.

#### Technical debt note
Less.
